bazel-mode.el

```diff
@@ -0,0 +1,167 @@
+;;; bazel-mode.el --- Emacs major mode for editing Bazel BUILD and WORKSPACE files
```

Comment:
Please enable lexical binding: https://www.gnu.org/software/emacs/manual/html_node/elisp/Using-Lexical-Binding.html

Reply:
Done.

Reply:
`lexical-binding` needs to be in the very first line of the file. You can use <kbd>M-x add-file-local-variable-prop-line RET lexical-binding RET t RET</kbd> to add it in the correct position.

Reply:
Done. I tried it there originally but this caused checkdoc to complain.